nginx/nginx.conf

```diff
@@ -23,7 +23,7 @@ http {
   add_header Strict-Transport-Security 'max-age=31536000; includeSubDomains; preload';
 
   # CSP
-  add_header Content-Security-Policy "default-src 'self'; script-src 'self' 'unsafe-inline' data: https://www.googletagmanager.com https://www.google-analytics.com; img-src 'self' https://www.google-analytics.com; style-src 'self'; font-src 'self'; object-src 'none'; frame-src ##APP_SERVER## https://www.youtube.com/; connect-src ##APP_SERVER## ##HMDA_API_SERVER## https://ffiec-api.cfpb.gov https://www.google-analytics.com https://s3.amazonaws.com;";
+  add_header Content-Security-Policy "default-src 'self'; script-src 'self' 'unsafe-inline' data: https://www.googletagmanager.com https://www.google-analytics.com; img-src 'self' data: https://www.google-analytics.com; style-src 'self'; font-src 'self' data:; object-src 'none'; frame-src https://www.youtube.com/; connect-src 'self' https://www.google-analytics.com https://s3.amazonaws.com;";
```

Comment:
The connect-src is fine here for the `homepage`, but it will need to be expanded for any of our apps that touch the api.

Reply:
For `v2` we'll be using calls like `/v2/public/...`, from the same domain. The `ffiec-api` subdomain will go away.